README.md

```diff
@@ -1,37 +1,173 @@
-# Spyder-Website
+# Spyder Website
 
-Minimal static [SvelteKit](https://kit.svelte.dev/) set-up made deployable to [GitHub Pages](https://metonym.github.io/sveltekit-gh-pages/). Based on [sveltekit-gh-pages](https://metonym.github.io/sveltekit-gh-pages/)
+![Spyder Logo](static/assets/media/spyder-logomark.png)
```

Comment:
We should use the [Readme banner here](https://github.com/spyder-ide/spyder-docs/blob/master/doc/_static/images/spyder_readme_banner.png) that is much better proportioned and explicitly designed as a banner, and IMO as a banner it would look better above the main title.